frontend/src/main-components/InsideFolderContent.js

```diff
@@ -1,22 +1,60 @@
 import React from "react";
 import styled from "@emotion/styled";
-
+import Flashcard from "../flashcards/Flashcard.js";
+import NewCard from "../flashcards/NewCard.js";
 
 const InsideFolderContent = (props) => {
   const Container = styled.div`
-    flex: 9;
+    flex: 9 1 auto;
     display: flex;
     border: 1rem solid white;
     border-radius: 1rem;
     background-color: white;
     margin: 1%;
+    flex-direction: column;
   `;
-  
 
+  //Ideally we would use a servlet to get cards, but for now here's an empty card container.
+  let flashcards;
+  // Currently we don't have a way to parse the folder key from the url
```

Comment:
You should label this as a TODO and this is an example that I found that might help. https://ui.dev/react-router-v4-query-strings/

Reply:
Cool, thanks!